locales/en-US/strings.properties

```diff
@@ -110,7 +110,22 @@ time_label_minute={number}m
 time_label_hour={number}h
 time_label_day={number}d
 
-# LOCALIZATION NOTE (settings_pane_*): This is shown in the Settings Pane sidebar.
+# LOCALIZATION NOTE (prefs_*, settings_*): These are shown in about:preferences
+# for a "Firefox Home" section. "Home" should be localized matching the sidebar.
```

Comment:
Let's add at the end: ", while Firefox should be treated as a brand and kept in English"

BTW, where is the string for the sidebar? There is no pane for the home page at this point

Reply:
I guess the answer to the latter is in

> …  although there are others that would be needed as part of mozilla-central

Reply:
Yeah, I suppose it's confusing in that the mozilla-central string doesn't exist yet, so I tried to be a bit more descriptive:
```
# LOCALIZATION NOTE (prefs_*, settings_*): These are shown in about:preferences
# for a "Firefox Home" section. "Firefox" should be treated as a brand and kept
# in English, while "Home" should be localized matching the about:preferences
# sidebar mozilla-central string for the panel that has preferences related to
# what is shown for the homepage, new windows, and new tabs.
```